packages/shared/src/utils/config/config.ts

```diff
@@ -71,32 +71,32 @@ export const getAppId = () => {
 };
 
 export const getSocketURL = () => {
-    const local_storage_server_url = window.localStorage.getItem('config.server_url');
+    const local_storage_server_url: string | null = window.localStorage.getItem('config.server_url');
     if (local_storage_server_url) return local_storage_server_url;
 
     let active_loginid_from_url;
```

Comment:
```suggestion
    let active_loginid_from_url: string | null = null;
```
@niloo-fs All other declared types can be removed from this `getSocketURL()` function because they are inferred successfully, so it's unnecessary to declare them. Please revert `line 74, line 78, line 80, line 84, line 85 & line 87.`  🙂